server/src/controllers/Account.controller.js

```diff
@@ -0,0 +1,59 @@
+import moment from 'moment';
+import dummyData from '../dummyDb/db';
+import Account from '../models/Account.model';
+
+const { accounts } = dummyData;
+/**
+ * @class AccountController
+ */
+export default class AccountController {
+  /**
+   * @method createAccount
+   *
+   * @param {Object} req
+   * @param {Object} res
+   *
+   * @returns {object} status code and data message
+   */
+  static createAccount(req, res) {
+    const { type } = req.body;
+    const {
+      id, firstName, lastName, email
+    } = req.decoded;
+    if (req.decoded.type === 'client') {
+      const newAccount = new Account();
+      const accountsLength = accounts.length;
+      const lastId = accounts[accountsLength - 1];
```

Comment:
I'm not sure I agree with this logic. From your code, `lastId` will be an object i.e. the account object not the id. I believe it should be
```suggestion
      const lastId = accounts[accountsLength - 1].id;
```

Reply:
done...